README.md

```diff
@@ -45,45 +45,107 @@ ReactDOM.render(
 
 For an interactive demo of all components, see https://balena-io-modules.github.io/rendition
 
-- [Alert](#alert)
-- [ArcSlider](#arcslider)
-- [Badge](#badge)
-- [BadgeSelect](#badgeselect)
-- [Banner](#banner)
-- [Box](#box)
-- [Button](#button)
-- [ButtonGroup](#buttongroup)
-- [Card](#card)
-- [CodeWithCopy](#codewithcopy)
-- [Container](#container)
-- [Divider](#divider)
-- [DropDownButton](#dropdownbutton)
-- [Filters](#filters)
-- [Fixed](#fixed)
-- [Flex](#flex)
-- [Form](#form)
-- [Gauge](#gauge)
-- [Heading](#heading)
-- [Img](#img)
-- [Input](#input)
-- [Link](#link)
-- [Markdown](#markdown)
-- [Mermaid](#mermaid)
-- [Modal](#modal)
-- [Navbar](#navbar)
-- [Pager](#pager)
-- [Pill](#pill)
-- [ProgressBar](#progressbar)
-- [Search](#search)
-- [Select](#select)
-- [Swatches](#swatches)
-- [Table](#table)
-- [Tabs](#tabs)
-- [Terminal](#terminal)
-- [TextWithCopy](#textwithcopy)
-- [Textarea](#textarea)
-- [Tooltips](#tooltips)
-- [Txt](#txt)
+- [Rendition](#rendition)
```

Comment:
How did this change?

Reply:
Ah, damn. It was my prettier.

Reply:
Actually, I'm not so sure really. Must be one of my VSCode extensions.

Reply:
@dimitrisnl I think you should run `build:docs` instead of manually editing the README. There is a README template in `docs` that you should modify, and the README should be automatically generated.